Sources/Collections/ObservableArray.swift

```diff
@@ -253,6 +253,10 @@ extension ObservableArray: DataSourceProtocol {
   public func numberOfItems(inSection section: Int) -> Int {
     return count
   }
+
+  public func item<Item>(at index: Int) -> Item {
```

Comment:
Item should be set to the type of array elements. You can check type name from the array declaration:

```swift
class ObservableArray<Item>
```

So, it's also `Item`. 

You don't have to explicitly set type aliases, just implement the protocol method and compiler will automatically infer associated types. 

```swift
public func item(at index: Int) -> Item {
  return self[index]
}
```

For the Array you'd should do

```swift
public func item(at index: Int) -> Iterator.Element {
  return self[index]
}
```

Because Iteraror.Element is the type that represent swift array elements.